### DISCUSSION TITLE
Srinidhi Resume

### DISCUSSION THREAD

**Discussion by srinidhifd** [recodehive/Stackoverflow-Analysis#352] [Q&A] [answered]
Linkedin ID: [Srinidhi Kulkarni](https://www.linkedin.com/in/srinidhikulkarni/)
Github ID : [Srinidhifd](https://github.com/srinidhifd)

[SrinidhiKulkarni-Frontend.pdf](https://github.com/user-attachments/files/16324901/SrinidhiKulkarni-Frontend.pdf)

I am interested in Frontend Developer Role / Web Developer / Full Stack Role, Please review my resume @sanjay-kv Sir.

**Comment by sanjay-kv** [comment]
[sri.pdf](https://github.com/user-attachments/files/16342961/sri.pdf)
[ATS Version of CV (12).docx](https://github.com/user-attachments/files/16342962/ATS.Version.of.CV.12.docx)

**Comment by srinidhifd** [comment]
@sanjay-kv Sorry sir for late reply was busy in semester exams couldn't look in this . It would be great if you could suggest what changes can be done to improve this resume which I am sharing now updated one . or if possible share me docx version of it with changes done .It would be very helpful for me.

[SrinidhiKulkarni-Resume.pdf](https://github.com/user-attachments/files/16624656/SrinidhiKulkarni-Resume.pdf)

**Comment by Jisha-tr** [comment, marked as answer]
Resume seems good let me know any help needed in peer review

**Reply by sandun-harshana** [reply]
Resume is nice and simple.

**Comment by SuBroXz11** [comment]
Resume is nice and simple.. Just add a short summary at the beginning

**Reply by srinidhifd** [reply]
@SuBroXz11 Thank you , will surely add short summary in my latest resume .

**Comment by BernaolaBarrientos** [comment]
Looks neat and focused — great work on keeping it concise.

**Comment by saif-pain** [comment]
It is good - Best of luck

**Comment by Sumit6569** [comment]
best of luck

**Comment by HARIHARANS24** [comment]
Resume is nice and simple.. Just add a short summary at the beginning

**Comment by srinidhifd** [comment]
Thank you all for the suggestions!

**Comment by sandun-harshana** [comment]
Resume is nice and simple.

**Comment by umidjon0211** [comment]
The resume seems good, let me know any help needed in peer review

**Reply by umidjonnajot5910** [reply]
goood

**Comment by Flauntleroy** [comment]
Good

**Comment by sam-dev-161127** [comment]
Please star mine repo here is it, https://github.com/Sam-Dev-161127/Nexaura. You can also start my other repo
